website/docs/r/api_key.html.markdown

```diff
@@ -0,0 +1,87 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: api_key"
+sidebar_current: "docs-mongodbatlas-resource-api-key"
+description: |-
+    Provides a API Key resource.
+---
+
+# Resource: mongodbatlas_api_key
+
+`mongodbatlas_api_key` provides a Project resource. This allows project to be created.
+
+~> **IMPORTANT WARNING:**  Changing the name of an existing Project in your Terraform configuration will result the destruction of that Project and related resources (including Clusters) and the re-creation of those resources.  Terraform will inform you of the destroyed/created resources before applying so be sure to verify any change to your environment before applying.
+
+## Example Usage
```

Comment:
Can we make sure to include example of each type of Endpoint with this resource: 
1. Organization API Key Endpoints
2. Organization API Key Access List Endpoints
3. Organization API Keys on Projects Endpoints

See more details in docs here: https://www.mongodb.com/docs/atlas/reference/api/apiKeys/

Reply:
resolving as Organization API Key Endpoints are the only ones available via Atlas Admin API at this time